multi-agent/lib/mas/elements/nodes/common/base_node.py

```
@@ -29,10 +29,11 @@ class MyNode(SomeMixin, OtherMixin, BaseNode): ...
     READS: ClassVar[set[str]] = set()
     WRITES: ClassVar[set[str]] = set()
 
-    def __init__(self, *, retries: int = 1, **kwargs: Any):
+    def __init__(self, *, retries: int = 1, tracing_service=None, **kwargs: Any):
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)